scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,190 @@
+@import 'settings';
+
+@mixin divider-position($divider-order, $large-screen: false) {
```

Comment:
Please add come comment explaining what this mixin calculates, it's a bit hard to understand it without context.

Reply:
Is this mixin needed at all?

As far as I can see every time it's used, it's used in different combination of arguments, and every combination of different arguments results in separate conditional if statement.

So for example, instead of 
```
    @include divider-position(1) {
      grid-row: 2;
    }
```
you could just write:

```
     &.p-equal-height-row--has-1st-divider .p-equal-height-row__col::before {
        @content;
      }
``` 

So, while I understand that it does abstract the positioning of divider into one place, it doesn't seem to remove any duplication or anything. It actually seems there is more code now, because of all the nested conditional statements inside this mixin?

Reply:
My aim for moving the code into a `mixin` is more for readability improvements and not necessarily to reduce code. I find having all the divider styles sitting inside `.p-equal-height-row` makes rest of the styling difficult to read. Maybe we can improve the mixin naming to provide better clarity?